conduct.md

```diff
@@ -52,35 +52,35 @@ TODO: Write an introductory paragraph.
 
 ### Communication
 
-TODO: Write an introductory paragraph.
+Communication is key to both successful and sustainable code. However, it is important that all communication between students and team members follows the three pillars of Respect, Trust, and Humility. In general, students will *respect* their peers, *trust* their contributions and thoughts, and practice *humility* in all aspects of communication, in person, on GitHub, in the Discord, and otherwise.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should communicate in a professional and respectful manner.
+- Students should use words like "Please" and "Thank you".
+- Students will allow other members and teams to speak freely.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should not communicate in a way that offends or disrespects others.
+- Students should not interrupt others--we all have something to share.
+- Students should not show signs of passive aggressive behavior.
 
 ### Participation
 
-TODO: Write an introductory paragraph.
+Participation for students will be based on a multitude of factors. We all have bad days and we all have topics we know more or less about, but how many times a student speaks a day will not solely make up their participation. It is being prepared each day, having done the prerequisite work, coming to class with an open mind to learn, and making an effort to be apart of either small group or large group conversations that will reflect students participation.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students will respect all peers' opinions.
+- Students will show up to every class ready to learn and engage with the class.
+- Students will evaluate their own participation and the participation of their group members by completing the participation surveys at the end of each week.
```

Comment:
Quick suggestion for this section:

I think it would be nice to have a point on fairly splitting up work on group assignments. This will ensure that the majority of the work is not assigned to one person and  that other members contribute to the group effort.

If this is already addressed or mentioned in a different section, please disregard this comment.